.gitignore

```diff
@@ -39,3 +39,6 @@ dist
 # Signing
 gpg.key
 cmd/trivy/trivy
+
+# bin folder
+bin/
```

Comment:
Since the Trivy project doesn't have `bin/`, I'd suggest excluding it on your end.
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
While Trivy itself doesn't have that directory, it's was created to contain `golangci-lint` when I ran the `mage` lint related commands hence I added it to `.gitignore`.
Isn't that a good reason ?
That said, thanks for the excellent pointers, I learned something new.

Reply:
I see. It's supposed to be installed under `$GOPATH/bin`.
https://github.com/aquasecurity/trivy/blob/b76a7250912cfc028cfef743f0f98cd81b39f8aa/magefiles/magefile.go#L25-L26

I guess you don't have `$GOPATH`, then `$PWD/bin` was created. It's sort of a bug in our Magefile, and not intended. In your environment, does `go env GOPATH` return something?

Reply:
So something is a bit strange here: `echo $GOPATH` returns empty but if I call `go env GOPATH` then it points to `/Users/username/go` however there's no `bin` folder in there.

Reply:
If I remember correctly, the default GOPATH is baked in Go so that Go can work without the `$GOPATH` env. We should fix the magefile. So, we don't need to add `bin/` in .gitignore since it's not intended.

Reply:
Fair enough, I have dropped the related commit.